diff --git a/node_modules/qrcode-terminal/lib/main.js b/node_modules/qrcode-terminal/lib/main.js
index 488cc1a..7404a73 100644
--- a/node_modules/qrcode-terminal/lib/main.js
+++ b/node_modules/qrcode-terminal/lib/main.js
@@ -1,7 +1,7 @@
 var QRCode = require('./../vendor/QRCode'),
     QRErrorCorrectLevel = require('./../vendor/QRCode/QRErrorCorrectLevel'),
-    black = "\033[40m  \033[0m",
-    white = "\033[47m  \033[0m",
+    black = "\0o33[40m  \0o33[0m",
+    white = "\0o33[47m  \0o33[0m",
     toCell = function (isBlack) {
         return isBlack ? black : white;
     },
